src/Bundle/test/app/AppKernel.php

```diff
@@ -22,29 +22,39 @@ class AppKernel extends Kernel
      */
     public function registerBundles()
     {
-        return [
+        if ('prod' === $this->getEnvironment()) {
+            $loader = require __DIR__ . '/../../../../vendor/autoload.php';
+            $loader->addPsr4('AppBundle\\', __DIR__ . '/../src/AppBundle/');
```

Comment:
Is it needed? `autoload-dev` in `composer.json` should already do the job.

Reply:
Nope, it's removed on "prod" env.

Reply:
When do we run the test application in `prod` environment?